core/src/main/java/org/apache/iceberg/actions/SizeBasedFileRewritePlanner.java

```diff
@@ -174,7 +174,7 @@ protected Iterable<List<T>> planFileGroups(Iterable<T> tasks) {
   }
 
   protected boolean enoughInputFiles(List<T> group) {
-    return group.size() > 1 && group.size() >= minInputFiles;
+    return group.size() > minInputFiles;
```

Comment:
How do we ensure that `minInputFiles > 1`?

Reply:
We ensure `minInputFiles > 0`.

```java
  private int minInputFiles(Map<String, String> options) {
    int value = PropertyUtil.propertyAsInt(options, MIN_INPUT_FILES, MIN_INPUT_FILES_DEFAULT);
    Preconditions.checkArgument(
        value > 0, "'%s' is set to %s but must be > 0", MIN_INPUT_FILES, value);
    return value;
  }
```